src/client/North/Form/NorthForm.jsx

```diff
@@ -10,7 +10,7 @@ import OIbForm from '../../components/OIbForm/OIbForm.jsx'
 import ApiSchemas from '../Apis.jsx'
 
 const NorthForm = ({ application, applicationIndex, onChange }) => {
-  const { api, applicationId } = application
+  const { id, api, applicationId } = application
   // Create the sections for the api (for example application.Link) for application not yet initialized
```

Comment:
if the id is given, do we need to pass also the applicationId? 
In term of naming, the applicationId should likely be applicationName and the id should be applicationId

Reply:
In this case the `applicationId` is used for the breadcrumb, to see which north application are we editing. Regarding the naming, yes, I think so. But it would be a huge changeset to rename all the occurrences in the project. Should I apply those changes?

Reply:
I leave it @burgerni10 to decide.

Reply:
I think we can keep "id" like that and rename  applicationId to "name" to have consistency across south and north connectors.
You can apply those changes in the whole code.

Reply:
Ok. So both `applicationId` and `dataSourceId` would be `name`? Wouldn't be cleaner if we leave the application and dataSource keyword there (like `applicationName` and `dataSourceName` suggested by @jfhenon )?. The data structure is different anyway and would be easier to make difference between north and south names in the code.

Reply:
I think id and name are better since they have the same purposes for both South and North connectors. We usually name the north connector "application" and the south connector "dataSource" anyway. So it would still be clear in the code since we will have application.name or dataSource.id

Reply:
I see, anyway it is a clean coding pattern to not have redundant attribute names, that can be found out from the main object name. Alright, I'll proceed